jenkins-centos7-atomicapp/artifacts/kubernetes/jenkins-rc.json

```diff
@@ -0,0 +1,47 @@
+{
+  "kind": "ReplicationController",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "jenkins"
+  },
+  "spec": {
+    "replicas": 1,
```

Comment:
Even replica count can be taken as env var, its always good make the manifests as much parameterized  as possible. Guess you will not have to edit the manifest for Jenkins RC if you want replica count to 2.